README.md

```diff
@@ -885,6 +885,8 @@ build all "tracked" content that has the status `NEEDS_BUILD`.
 
 > To re-run failed builds, use `rsconnect content build run --retry`. This will build
 all tracked content in any of the following states: `[NEEDS_BUILD, ABORTED, ERROR, RUNNING]`.
+>
+> If you encounter an error stating that a build operation is already running, you can use `rsconnect content build run --force`. This will override this check and build any content marked as `NEEDS_BUILD`.
```

Comment:
Should this include something like, "make sure any previously launched `content build` operation is no longer active"?

Reply:
Ahh yeah that's a good suggestion. Added a note for that and made an attempt to improve the flow of the wording as well